tests/language-feature-scripts/access-interceptor-scope-localizer-allow-use-variadic-notation.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+Verifies that generated access interceptors disallow protected property direct write
```

Comment:
This should not be a `.phpt` test: instead, a functional test should be written for each of the tests in this dir: https://github.com/Ocramius/ProxyManager/tree/917bf1698243a1079aaa27ed8ea08c2aef09f4cb/tests/ProxyManagerTest/Functional